src/plugins/data/public/ui/filter_bar/filter_editor/lib/filter_label.test.tsx

```diff
@@ -1,143 +0,0 @@
-/*
```

Comment:
- [ ] why it was removed?

Reply:
sorry, it's correct. For some reason it was marked as removed instead of moved 

<img width="296" alt="image" src="https://user-images.githubusercontent.com/20072247/159646307-79d0acc6-c08c-4968-889f-7a2c441b6bf5.png">